vault/102/README.md

```diff
@@ -0,0 +1,60 @@
+# <a href="http://www.devopsloft.io">DevOps Loft</a>
+
+<img src="http://www.devopsloft.io/static/logo.png" alt="drawing" width="250" hight="250"/>
+
+## Disclaimer: AWS services are subject to charges. Be aware and use cautiously
+
+<details>
+  <summary>Prerequisites</summary>
+  <ul>
+    <li>Install Git</li>
+    <li>GitHub account</li>
+    <li>Fork this repo</li>
+    <li>Clone your forked repo</li>
+    <li>Install Vagrant</li>
+    <li>Install VirtualBox</li>
+    <li>AWS account</li>
+    <li>AWS cli credentials set in ~/.aws</li>
+    <li>S3 bucket</li>
+  </ul>
+</details>
+
```

Comment:
I think it will be better to put here kind of title. Otherwise it seems as it is part of prerequisite

Reply:
+1